File: CS/9.1/WinWebSolution.Win/WinApplication_E859.cs
using System;
using DevExpress.ExpressApp;
using WinWebSolution.Module;

namespace WinWebSolution.Win {
    partial class WinWebSolutionWindowsFormsApplication {
        protected override CollectionSourceBase CreateCollectionSourceCore(ObjectSpace objectSpace, Type objectType, string listViewID) {
            CollectionSourceBase cs = LinqCollectionSourceProvider.GetCollectionSource(objectSpace, objectType, listViewID);
            if (cs == null) {
                cs = base.CreateCollectionSourceCore(objectSpace, objectType, listViewID);
            }
            return cs;
        }
    }
}

File: VB/VB/WinWebSolution.Win/WinApplication_E859.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports WinWebSolution.Module

Namespace WinWebSolution.Win
	Partial Public Class WinWebSolutionWindowsFormsApplication
		Protected Overrides Function CreateCollectionSourceCore(ByVal objectSpace As ObjectSpace, ByVal objectType As Type, ByVal listViewID As String) As CollectionSourceBase
			Dim cs As CollectionSourceBase = LinqCollectionSourceProvider.GetCollectionSource(objectSpace, objectType, listViewID)
			If cs Is Nothing Then
				cs = MyBase.CreateCollectionSourceCore(objectSpace, objectType, listViewID)
			End If
			Return cs
		End Function
	End Class
End Namespace